examples/audio.yml

```diff
@@ -26,42 +26,3 @@
     src: null
     cover:
       orientation: "landscape"
--
-  _valid: false
-  _name: "audio-without-id"
-  _data:
-    type: "audio"
-    html: '<audio></audio>'
--
-  _valid: false
-  _name: "valid-audio-without-type"
```

Comment:
Moved from here to `contentblock` since after dereferencing they need the full content block definition for validation to work correctly